source/renderer/app/components/wallet/WalletCreateDialog.js

```diff
@@ -211,9 +211,19 @@ export default class WalletCreateDialog extends Component<Props, State> {
     const { isSubmitting } = this.state;
     const dialogClasses = classnames([styles.component, 'WalletCreateDialog']);
 
+    const walletNameField = form.$('walletName');
+    const spendingPasswordField = form.$('spendingPassword');
+    const repeatedPasswordField = form.$('repeatPassword');
+
+    const formValuesNotSet =
+      !walletNameField.value ||
+      !spendingPasswordField.value ||
+      !repeatedPasswordField.value;
+
```

Comment:
@DeeJayElly Please change variable name to `canSubmit` or something similar. Also, `isSubmitting` clause inside as well as filed errors check (you can check in form validator) e.g. `!isSubmittings || !walletNameField.error...`

Reply:
Done